tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -6,16 +6,23 @@ import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import type {PersonalDetails} from '@src/types/onyx';
 import type Report from '@src/types/onyx/Report';
+import type {OptionData} from '@libs/ReportUtils';
+import createRandomOptionData from 'tests/utils/collections/optionData';
```

Comment:
Are you sure this path it correct? It should not be  `../utils/collections/optionData`?